ansible/roles/stack-sunbird/templates/sunbird_nodebb.env

```diff
@@ -1,10 +1,15 @@
 # Initialization variable for mongo
 # Ref: https://community.nodebb.org/topic/4325/set-up-nodebb-via-environment-variables
-database=mongo
+database=redis
+# redis database number
+redis_database={{nodebb_redis_database|default('0')}}
+# declearing js heap memory size
+nodebb_heap_size={{nodebb_heap_size|default('2048')}}
```

Comment:
the left hand side of this should the variable name used for nodejs heap memory - `NODE_OPTIONS`
Or we can also add this value as a env in deployment manifest as
```
env:
  name: NODE_OPTIONS
  value: XXX
```

Reply:
changed, you can see the latest changes now